### DISCUSSION TITLE
Bug in rendering mathematical formulas having fractions

### DISCUSSION THREAD

**Discussion by abzokhattab** [requarks/wiki#5067] [Error / Bug Report]
Hello team, 
There is a bug in rendering mathematical formulas having fractions using the markdown editor 

Here is how to reproduce: 
1. Create a new page and use markdown formatting.
2. Use  the `$\frac {} {} $` operator. for example `$\frac{b }{c}$` 
3. The formula will be rendered correctly in the preview tap, but after saving and creating the page, this fraction will not get rendered correctly (refer to the screenshots).

![image](https://user-images.githubusercontent.com/59809993/156340060-b4061008-cb3f-4541-adac-55bc0adc1a65.png)

![image](https://user-images.githubusercontent.com/59809993/156340076-55bde2f6-7c69-420b-a876-84dcdd302675.png)

**Comment by abzokhattab** [comment]
any updates on this issue?

**Comment by jake-pfannenstiel** [comment]
For inline MathJax, using one $ for beginning and end, there needs to be text after the second $.  There are some better explanations in the discussions
